docs/AA--Javascript-in-EE/value-objects/label.md

```diff
@@ -0,0 +1,168 @@
+## Label Value Object
+The label value object represents a label which consists of a singular string and plural string.  For example a shoe label where singular is `shoe` and plural is `shoes`.
+
+## Features
+
+The label value object exposes the singular and plural forms of the labels as their raw value but also allows for retrieving formatted strings via helper methods.
+
+## Installation/Loading
+
+The label vo is available via the `eejs` global on its `valueObjects` property.  So you can access it directly via `eejs.valueObjects.Label` provided the bundled `@eventespresso/vo` javascript has been loaded before your script. For instructions on different ways to do this you can [go here](README.md#Usage).
```

Comment:
since "vo" is an acronym, it should be capitalized. 
As well, before using an acronym for the first time on its own, it should first be defined by introducing it after the full term that it represents. So for example, in the "Features" section above this, we would do something like the following:

> The label value object (VO) exposes the singular and plural forms...

which then allows us to use the acronym on its own since we have now defined it:

> The label VO is available...